packages/connectivity/example/test_driver/connectivity.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:connectivity/connectivity.dart';
+
+void main() {
+  final Completer<String> completer = Completer<String>();
+  enableFlutterDriverExtension(handler: (_) => completer.future);
+  tearDownAll(() => completer.complete(null));
+
+  group('Connectivity test driver', () {
```

Comment:
Elsewhere in the plugins repo we use the stringified class name '$Connectivity' to identify group so you may want to be consistent with that, though it's not required. I wouldn't say "test driver" though.